src/rules/extensions.js

```diff
@@ -64,13 +64,23 @@ module.exports = {
       return resolvedFileWithoutExtension === resolve(file, context)
     }
 
+    function isExternalRootModule(file) {
+      if (!isExternalModule(file, context,resolve(file, context))) return false
+      if (file.indexOf('/') >= 0) return false
```

Comment:
Nice catch, totally forgot and I actually using private modules! Should be handled properly now.